ruby/secret_agents.rb

```diff
@@ -0,0 +1,36 @@
+#Encrypt method: move each letter in a string one letter forward
+
+def encrypt(string)
+  index =  0
+  while index < string.length
+    string[index] = string[index].next!
```

Comment:
This fails on "z" letters. Either change the method to account for the "z" case or manually change "z" to "a" with an if statement.